pangeo_forge_runner/commands/expand_meta.py

```diff
@@ -19,9 +21,17 @@ class ExpandMeta(BaseCommand):
     flags = common_flags
 
     def start(self):
-        with tempfile.TemporaryDirectory() as d:
-            self.fetch(d)
-            feedstock = Feedstock(Path(d) / self.feedstock_subdir)
+        if os.path.exists(self.repo):
```

Comment:
Duplication of similar code in `Bake.start()`

Reply:
@derekocallaghan hmm, github seems to be showing a stale version of this here - I had moved these up to the base's `fetch` to avoid this duplication

Reply:
Ah, I think I was looking at the oldest commit, apologies. I see it in the base `fetch()` now.